server.js

```diff
@@ -13,8 +13,8 @@ app.post("/api/upload", async (request, response) => {
     const uploadedResponse = await cloudinary.uploader.upload(fileStr, {
       upload_preset: "upload",
     });
-    console.log(uploadedResponse);
-    response.status(200).send(uploadedResponse);
+    console.log(uploadedResponse.secure_url);
```

Comment:
do you need this one? secure.url looks like something you dont want to log in the console 👩‍🎤

Reply:
i need this for later . i console logged it so i dont forget it 🙈

Reply:
nagut 🙃